src/test/java/org/jenkinsci/plugins/workflow/libs/LibraryStepTest.java

```diff
@@ -85,7 +85,11 @@ public class LibraryStepTest {
             null, null, Collections.<GitSCMExtension>emptyList())));
         s.setChangelog(false);
         r.assertEqualDataBoundBeans(s, stepTester.configRoundTrip(s));
-        snippetizerTester.assertRoundTrip(s, "library changelog: false, identifier: 'foo@master', retriever: legacySCM([$class: 'GitSCM', branches: [[name: '${library.foo.version}']], extensions: [], userRemoteConfigs: [[url: 'https://nowhere.net/']]])");
+        if (r.jenkins.get().getPlugin("git").getWrapper().getVersion().startsWith("4.")) {
+            snippetizerTester.assertRoundTrip(s, "library changelog: false, identifier: 'foo@master', retriever: legacySCM([$class: 'GitSCM', branches: [[name: '${library.foo.version}']], extensions: [], userRemoteConfigs: [[url: 'https://nowhere.net/']]])");
+        } else {
+            snippetizerTester.assertRoundTrip(s, "library changelog: false, identifier: 'foo@master', retriever: legacySCM(scmGit(branches: [[name: '${library.foo.version}']], extensions: [], userRemoteConfigs: [[url: 'https://nowhere.net/']]))");
```

Comment:
I guess this is not simply `git` because that would conflict with the `Step`? :disappointed:

Reply:
That's correct.  I wanted to use `git` but there is already a `git` step defined.